wiki/Registration/id.md

```diff
@@ -1,43 +1,23 @@
----
-outdated: true
----
-Bagaimana cara saya mendaftar?
-------------------------------
+# Pendaftaran
 
-<center>
-**<font size="3">Tolong diingat bahwa kamu hanya boleh mempunyai satu akun! 2 akun atau lebih adalah pelanggaran keras dari <span class="plainlinks"> [peraturan osu!](/wiki/Rules)</span>. Ikuti langkah di bawah ini untuk membuat akun osu!mu.**</font>
-<span style="font-size:18px">1.Pergi ke <span class="plainlinks"> [website](https://osu.ppy.sh)</span> dan klik "<b>I'm new!</b>" untuk melanjutkan ke formulir pendaftaran. </span>
+*Harap diperhatikan bahwa memiliki lebih dari satu akun osu! merupakan salah satu pelanggaran terhadap [peraturan osu!](/wiki/Rules/).*
 
-<span style="font-size:18px">2. Isilah formulir di bawah ini kemudian tekan "Continue" </span>
+1. Setelah [menginstal](/wiki/installation) osu! dan menjalankannya, osu! akan meminta Anda untuk masuk atau mendaftar. Dengan asumsi bahwa Anda memerlukan akun untuk bisa masuk, klik `Create an account` dan biarkan osu! tetap terbuka. Seharusnya Anda akan diarahkan ke halaman [Create Account](https://osu.ppy.sh/p/register).
```

Comment:
` ... diarahkan ke halaman [Create Account](https://osu.ppy.sh/p/register). ... ` 

Why use Create Account instead of `Registrasi Akun` ?

Reply:
This article is pointing out at the [old site](https://osu.ppy.sh).
That's why i'm not translating it (for now).